.rubocop.yml

```diff
@@ -1,80 +1,199 @@
-  - .rubocop.yml
-  - .codeclimate.yml
-  - .travis.yml
-  - .gitignore
-  - .rspec
-
-  - Gemfile.lock
-  - CHANGELOG.{md,markdown,txt,textile}
-  - CONTRIBUTING.{md,markdown,txt,textile}
-  - readme.{md,markdown,txt,textile}
-  - README.{md,markdown,txt,textile}
-  - Readme.{md,markdown,txt,textile}
-  - ReadMe.{md,markdown,txt,textile}
-  - COPYING
-  - LICENSE
-
-  - site/**/*
-  - test/**/*
-  - vendor/**/*
-  - features/**/*
+  - lib/jekyll/cleaner.rb
+  - lib/jekyll/collection.rb
+  - lib/jekyll/command.rb
+  - lib/jekyll/commands/build.rb
+  - lib/jekyll/commands/clean.rb
+  - lib/jekyll/commands/doctor.rb
+  - lib/jekyll/commands/help.rb
+  - lib/jekyll/commands/new.rb
+  - lib/jekyll/commands/serve
+  - lib/jekyll/commands/serve/servlet.rb
+  - lib/jekyll/commands/serve.rb
+  - lib/jekyll/configuration.rb
+  - lib/jekyll/converter.rb
+  - lib/jekyll/converters/identity.rb
+  - lib/jekyll/converters/markdown
+  - lib/jekyll/converters/markdown/kramdown_parser.rb
+  - lib/jekyll/converters/markdown/rdiscount_parser.rb
+  - lib/jekyll/converters/markdown/redcarpet_parser.rb
+  - lib/jekyll/converters/markdown.rb
+  - lib/jekyll/converters/smartypants.rb
+  - lib/jekyll/convertible.rb
+  - lib/jekyll/deprecator.rb
+  - lib/jekyll/document.rb
+  - lib/jekyll/drops/collection_drop.rb
+  - lib/jekyll/drops/document_drop.rb
+  - lib/jekyll/drops/drop.rb
+  - lib/jekyll/drops/jekyll_drop.rb
+  - lib/jekyll/drops/site_drop.rb
+  - lib/jekyll/drops/unified_payload_drop.rb
+  - lib/jekyll/drops/url_drop.rb
+  - lib/jekyll/entry_filter.rb
+  - lib/jekyll/errors.rb
+  - lib/jekyll/excerpt.rb
+  - lib/jekyll/external.rb
+  - lib/jekyll/filters.rb
+  - lib/jekyll/frontmatter_defaults.rb
+  - lib/jekyll/generator.rb
+  - lib/jekyll/hooks.rb
+  - lib/jekyll/layout.rb
+  - lib/jekyll/liquid_extensions.rb
+  - lib/jekyll/liquid_renderer/file.rb
+  - lib/jekyll/liquid_renderer/table.rb
+  - lib/jekyll/liquid_renderer.rb
+  - lib/jekyll/log_adapter.rb
+  - lib/jekyll/page.rb
+  - lib/jekyll/plugin.rb
+  - lib/jekyll/plugin_manager.rb
+  - lib/jekyll/publisher.rb
+  - lib/jekyll/reader.rb
+  - lib/jekyll/readers/collection_reader.rb
+  - lib/jekyll/readers/data_reader.rb
+  - lib/jekyll/readers/layout_reader.rb
+  - lib/jekyll/readers/page_reader.rb
+  - lib/jekyll/readers/post_reader.rb
+  - lib/jekyll/readers/static_file_reader.rb
+  - lib/jekyll/regenerator.rb
+  - lib/jekyll/related_posts.rb
+  - lib/jekyll/renderer.rb
+  - lib/jekyll/site.rb
+  - lib/jekyll/static_file.rb
+  - lib/jekyll/stevenson.rb
+  - lib/jekyll/tags/highlight.rb
+  - lib/jekyll/tags/include.rb
+  - lib/jekyll/tags/link.rb
+  - lib/jekyll/tags/post_url.rb
+  - lib/jekyll/theme.rb
+  - lib/jekyll/url.rb
+  - lib/jekyll/utils/ansi.rb
+  - lib/jekyll/utils/platforms.rb
+  - lib/jekyll/utils.rb
+  - lib/jekyll/version.rb
+  - lib/jekyll.rb
+  - bin/**/*
   - script/**/*
-  - spec/**/*
+  - vendor/**/*
+Lint/AmbiguousRegexpLiteral:
+  Exclude:
+  - features/step_definitions.rb
```

Comment:
You are actually working around a literal Ruby warning, so this error will still surface on other linters (like Atom Linter, Sublime Text Linter and others.) According to Cucumber themselves it seems the preferred way to fix this with them is to just use `%r!!` which works for all parties, it's less escaping!